applications/SystemIdentificationApplication/python_scripts/sensor_sensitivity_solvers/system_identification_static_analysis.py

```diff
@@ -36,7 +37,11 @@ def Initialize(self):
         model_part.ProcessInfo[KratosSI.ADAPT_PERTURBATION_SIZE] = sensor_settings["adapt_perturbation_size"].GetBool()
         self.listof_sensors = GetSensors(model_part, sensor_settings["list_of_sensors"].values())
 
-        self.measurement_residual_response_function = KratosSI.Sensors.MeasurementResidualResponseFunction()
+        aggregation_technique = sensor_settings["aggregation_technique"].GetString()
```

Comment:
I don't see how this `aggregation_technique` variable is used?

Reply:
it is left over, I will remove